### ISSUE TITLE
Discuss potential O'Reily "sponsorship"

### ISSUE THREAD

**Issue by dlindahl** [sacjs/admin#1]
I got the following message from Jason Yee from O'Reily via Meetup:

> Hi Derek,  Thanks for organizing the Sacramento JavaScript meetup. At O'Reilly we love communities and their power to change the world by sharing knowledge.
> 
> We'd like to help your community grow. Would you be interested in free videos, books, conference passes and resources from O'Reilly Media? How about sponsorship for an upcoming meetup?
> 
> If that sounds good, just go to our partner form at http://oreil.ly/partner to tell us a little more about your group and the best address to reach you to send any physical materials. After you complete the form, we'll work with you on how we can best help. We'll also send you occasional emails to let you know about tips, resources and conferences that we hope you'll find useful and share with your community.  
> 
> Thanks again and we look forward to partnering with your community.
> 
> Jason Yee
> jyee@oreilly.com

While generally I am open to any free stuff vendors want to give us, I'm a bit hesitant on any sponsorship that would lead a company to think they can take over a specific event.

I believe BrooklynJS has at least one sponsor per event that covers most of their operating costs, but they only allow a 2-minute block of time for the sponsor to speak at their events. I am definitely in favor of an approach like that.

My only other thought on the matter is that for an event like our first one where a company is the featured speaker, then having a sponsor may not make sense. We wouldn't want a sponsor to cover whatever costs we'd have for one of their competitors to speak :laughing: 

I'm going to reply to Jason and see what he has in mind. I'd love to hear any feedback from the rest of the @sacjs/staff

**Comment by samt** [comment]
SacJS has had a small relationship with O'Reilly for a while (back when we were doing basic meetups), but it was never a sponsorship. Right now, without the sponsorship, they will send us free stuff and some promo codes for massive discounts for conferences and books if we ask them.

I think it would be worth while to look into a full sponsorship and see what they offer; we absolutely are in no obligation to accept. They do treat their usergroups very well as their business model is building customers through tons of outreach.

**Comment by dlindahl** [comment]
I've registered for their partner platform and have been granted 3 codes for JS: The Good Parts ebook.

@samt I noticed that the O'Reily logo is already listed as a sponsor. That was one of their requirements, putting the logo on their. Did you do that? Just want to make sure I'm not losing my mind (anymore than usual)

As we grow, we should remember to reach out to them to cover any costs that may come up. Pizza, room rentals, etc. Once that becomes an actual problem, we should open a new Issue to discuss our options.
